app/models/question.rb

```diff
@@ -88,4 +92,43 @@ def sort_answers
       b.sort_value <=> a.sort_value
     end
   end
+
+  def create_action_verb
+    "Asked a Question"
+  end
+
+  def update_action_verb
+    "Updated his/her Question"
+  end
+
+  def tracking_information
+    [:id, :title, :content, :created_since]
+  end
+
```

Comment:
Can we move all these `#*_action_verb` methods to  a module and **mixin** to the models?